src/components/LabelBasicSelect/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Label from '../styled-components/Label';
+import BasicSelect from '../styled-components/BasicSelect';
+
+interface LabelBasicSelectProps {
+  label?: string;
+  text?: string;
+  value?: string;
+  onChange?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  options?: { value: string; label: string }[];
+  onBlur?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  hasError?: boolean;
+  errorMessage?: string;
+}
+
+const LabelBasicSelect = (props: LabelBasicSelectProps) => {
+  const { label, text, value, onChange, options, onBlur, hasError, errorMessage } = props;
+  return (
+    <div className='FormContainer'>
+      <Label htmlFor={label} text={text} />
+      <BasicSelect
+        value={value}
+        onChange={onChange}
+        options={options}
+        onBlur={onBlur}
+        hasError={hasError}
+      />
+      {hasError ? <span className='BasicInput__error'>{errorMessage}</span> : null}
```

Comment:
```suggestion
      {hasError && <span className='BasicInput__error'>{errorMessage}</span>}
```
&& 연산자로 동일한 효과를 내지만 가독성을 올리는 건 어떨까요